src/app/BasketProducts/index.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Button from 'src/stories/Button';
+
+export default function BasketProducts() {
+    return (
+        <div>
+            <Button label="Delete" />
```

Comment:
This "Button" component is implemented as an example in this project. Let's not use it. Use plain <button>.

Reply:
Done